cpp_utils/MMU.cpp

```diff
@@ -59,7 +59,7 @@ static uint32_t flashPageToOffset(uint32_t page) {
 	}
 	printf("\n");
 	printf("APP CPU MMU\n");
-	for (uint8_t i = 0; i < 256; i++) {
+	for (uint16_t i = 0; i < 256; i++) {
```

Comment:
One question, is it better to change value type or maybe its just a typo and should be `< 255`?